### PR TITLE
Test build multi-arch image in CI builds

### PR DESCRIPTION
**Pull request by utsavanand2**
Signed-off-by: Utsav Anand <utsavanand2@gmail.com>

## Description

Test building multi-arch images in CI stage for testing.

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

Check the build logs [here](https://github.com/utsavanand2/faas-cli/runs/1412905346?check_suite_focus=true)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -40,20 +40,20 @@ jobs:
...
           push: false
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
           target: release
           build-args: |
             VERSION=latest-dev
             GIT_COMMIT=${{ github.sha }}
             REPO_URL=https://github.com/${{ env.USER_REPO }}
-          tags: openfaas/faas-cli:${{ github.sha }}-root
+          tags: ghcr.io/openfaas/faas-cli:${{ github.sha }}
```
Comment (alexellis) [inline .github/workflows/build.yaml:56]: Was "openfaas/faas-cli" used anywhere in docker in scripts?

Comment (utsavanand2) [reply .github/workflows/build.yaml]: @alexellis I've changed it back to openfaas/faas-cli

Comment (utsavanand2) [reply .github/workflows/build.yaml]: @alexellis I've changed it back to openfaas/faas-cli

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**